spec/factories/school_types.rb

```diff
@@ -1,5 +1,5 @@
 FactoryBot.define do
   factory :school_type do
-    label { Faker::Lorem.unique.sentence }
+    label { Faker::Lorem.unique.sentence.gsub('.', '') }
```

Comment:
Seems unnecessary given that using lorem for setting school type is going to give nonsense anyway.

Reply:
Was just annoyed by the period getting in the way of my nice '<school type>, <religious character>' formatting.